platform/linux-generic/odp_system_info.c

```diff
@@ -471,7 +471,14 @@ uint64_t odp_sys_huge_page_size(void)
 
 static int pagesz_compare(const void *pagesz1, const void *pagesz2)
 {
-	return (*(const uint64_t *)pagesz1 - *(const uint64_t *)pagesz2);
+	uint64_t val1 = *(const uint64_t *)pagesz1;
+	uint64_t val2 = *(const uint64_t *)pagesz2;
+
+	if (val1 < val2)
+		return -1;
+	else if (val1 > val2)
```

Comment:
`else` is redundant. `val1`/`2` could be `const`.

Reply:
Fixed in V2.